### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Notebooks - Create_Keyboard: The user is unable to activate the 'Delete' button using keyboard space or return key associated with '+Code' button and '+Text' control.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19873]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG2.1.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'File' menu.
5. Tab to 'New Notebook' and select.
6. Tab to '+Code' button and select.
7. Tab to 'Delete' button.
8. Use keyboard to activate the delete button.

## Actual:
The user is unable to activate the 'Delete' button using keyboard space or return key associated with '+Code' button and '+Text' control.

## Expected:
The user should be able to activate the 'Delete' button using keyboard using keyboard space or return key.

## User Impact:
Keyboard users will difficulty in accessing the controls.

## Attachment:
![13_Delete button does not](https://user-images.githubusercontent.com/98393966/176432636-c5c91330-454b-4b5b-928b-013ded9ed8ee.gif)

**Comment by chlafreniere** [comment]
@bbarik06 if a user tabs until the "+" is highlighted, and then uses the arrow keys until hovering over the 'Delete' command, I was able to press either the space or enter button to delete the cell.

Is this sufficient (using arrow keys)?

**Comment by alanrenmsft** [comment]
please let us know if this is not sufficient, the issue is going out of SLA, so closing it.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.36.1
![image](https://user-images.githubusercontent.com/85863999/219282335-3a9b8a34-62c0-46c9-bdfa-98cd5557a90b.png)
